core/src/main/java/org/phoenixframework/liveview/lib/Document.kt

```diff
@@ -0,0 +1,153 @@
+package org.phoenixframework.liveview.lib
+
+
+class Document {
+    private var nativeObject: Long
+
+
+    constructor() {
+        nativeObject = empty()
+    }
+
+    internal constructor(pointer: Long) {
+        nativeObject = pointer
+    }
+
+    companion object {
+        /** Parses a `Document` from a string
+         * @throws Exception if the document is malformed
+         * */
+        @Throws
+        fun parse(string: String): Document {
+            val result = JavaResult(do_parse(string))
+            return result.document ?: throw Exception(result.error)
+        }
+
+        private external fun do_parse(text: String): Long
+
+        /** Output logs from the Rust side into android's logcat */
+        private external fun initialize_log()
+        enum class ChangeType {
```

Comment:
Leave a blank line between the `fun` and `enum class`

```suggestion
        private external fun initialize_log()
        
        enum class ChangeType {
```